### PR TITLE
Add edit network action in the Network module

### PR DESCRIPTION
**Pull request by aeddaqqa**
Add edit network action in the Network module and save the update in the local-storage

### CODE REVIEW

File: src/views/wallet/Transfer.vue [hunk 1]
```diff
@@ -121,7 +121,7 @@
                                 block
                                 :disabled="!canSendAgain"
                             >
-                                Start Again
+                                New Transaction
```
Comment (Chriserus) [inline src/views/wallet/Transfer.vue]: If it's not translated and you change such place - please add translation

**Review by Chriserus — CHANGES_REQUESTED** [review]

File: src/views/wallet/Transfer.vue (hunk 1 above)
Comment (aeddaqqa) [reply src/views/wallet/Transfer.vue]: no need for translation, this pr contains changes for camino-suite and the translation is not implemented yet

**Review by Chriserus — APPROVED** [review]
LGTM

**Review by Ayagoumi — CHANGES_REQUESTED** [review]
This PR broke the add network functionality.

**Review by Ayagoumi — APPROVED** [review]
LGTM

**Pull request merged**